include/deal.II/lac/lapack_templates.h

```diff
@@ -20,292 +20,294 @@
 #include <deal.II/base/config.h>
 #include <deal.II/lac/lapack_support.h>
 
+using dealii::BlasIntType;
```

Comment:
This brings the declaration into the global namespace into every file that `#include`s this header. Please use the qualified form.